app/src/main/java/com/f0x1d/logfox/ui/activity/MainActivity.kt

```diff
@@ -113,6 +114,9 @@ class MainActivity: BaseViewModelActivity<MainViewModel, ActivityMainBinding>(),
 
                 else -> getColor(R.color.navbar_transparent_background)
             }
+        } else if (gesturesAvailable && Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
`gesturesAvailable` is already `Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q`. Just add `SupressLint` annotation